src/tests/test_magic.py

```diff
@@ -747,15 +748,16 @@ def test_autopolars_infer_schema_length(ip):
 
 
 def test_mutex_autopolars_autopandas(ip):
+    ip.run_line_magic("config", "SqlMagic.autopolars = False")
```

Comment:
has default been changed to `True`?  We need to change [here](https://jupysql.ploomber.io/en/latest/api/configuration.html#autopandas)

Reply:
it hasn't. I added it to be more explicit